test/bacnet/basic/object/ai/src/main.c

```diff
@@ -30,6 +30,9 @@ static void testAnalogInput(void)
     BACNET_READ_PROPERTY_DATA rpdata;
 
     Analog_Input_Init();
+    len = Analog_Input_Count();
+    zassert_equal(len, 4, NULL);
```

Comment:
@Michail-Antropov The test needs to assert the code follows the preprocessor definitions for controlling size rather than hardcoding the default.